docs/coach/groups.rst

```diff
@@ -4,7 +4,7 @@
 Manage Groups
 -------------
 
-In case you need to further divide learners inside classes, for example to address the different progress needs, you can use the **Groups** feature. Create and delete groups, as well as assign learners to them from the **Groups** sub-tab in the **Plan** tab (|edit| icon) of your **Coach** dashboard. Default view displays the list of all groups for the selected class, with the number of assigned learners for each group.
+In case you need to further divide learners inside classes, for example to address the different progress needs, you can use the **Groups** feature. Create and delete groups, as well as assign learners to them from the **Groups** sub-tab in the |edit| **Plan** tab of your |coach| **Coach** dashboard. Default view displays the list of all groups for the selected class, with the number of assigned learners for each group.
```

Comment:
> `|edit| **Plan** tab`

FYI @radinamatic this is an example of what I was describing where the icon doesn't always map one-to-one to a label.

Reply:
Yes, we'll have to do some clean-up and eventual renaming before trying to achieve 1:1 consistency :+1: